test/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+tests(GROUPS fuzz
+      SOURCES
+      simplify.cpp
+)
+
+
+target_link_libraries(fuzz_simplify PRIVATE Halide::Halide)
+
+# Allow OSS-fuzz to manage flags directly
+if (DEFINED ENV{LIB_FUZZING_ENGINE})
```

Comment:
The `LIB_FUZZING_ENGINE` is a somewhat well-known variable. It's used to allow for overriding the fuzzing engine entirely. So the goal here is to have libfuzzer+address-sanitizer be the default but then if LIB_FUZZING_ENGINE is declared you can override the fuzzing engine to one of;
- [Centipede](https://github.com/google/centipede)
- [Hongfuzz](https://github.com/google/honggfuzz)
- [AFL++](https://github.com/AFLplusplus/AFLplusplus)
- etc.

Using this variable is also a [precursor to integrating with OSS-fuzz](https://google.github.io/oss-fuzz/advanced-topics/ideal-integration/#build-support). So while I could rename it to `HALIDE_LIB_FUZZING_ENGINE` it would probably add some unnecessary indirection further down the line.

Reply:
OK, cool. Please try to integrate these comments into the CMake file or whatnot so that they are captured in the source rather than just here in the PR -- it will make our lives easier in the future.